exist-core/src/test/xquery/xquery3/parseIetfDate.xqm

```diff
@@ -51,8 +51,10 @@ declare
     %test:args('') %test:assertError("FORG0010")
     %test:args('asdfa') %test:assertError("FORG0010")
 
-    %test:args(0) %test:assertError("FORG0010")
-    %test:args(1) %test:assertError("FORG0010")
+    %test:args(0) %test:assertError("XPTY0004")
```

Comment:
These test cases that will result in XPTY0004 can be removed. They just test the type system of exist and not the function itself.

Reply:
I believe this issue is now tested elsewhere enough.